tests/nemo_text_processing/en/data_text_normalization/test_cases_serial.txt

```diff
@@ -0,0 +1,29 @@
+t-0t25d12-f~t-zero t twenty five d twelve-f
+133-ABC~one hundred thirty three-ABC
+B2A23C~B two A twenty three C
+25d08A~twenty five d zero eight A
+C24~C twenty four
+W2s~W two s
+1-413-te-b-1-5~one-four hundred thirteen-te-b-one-five
+b-c-1-5-b-s-b~b-c-one-five-b-s-b
+MIG-25/235212-asdg~MIG-twenty five/two three five two one two-asdg
+1/f-4s~one/f-four s
+1/f~one per degree Fahrenheit
+4s~four seconds
+7-eleven~seven-eleven
+2x~two x
+11:30EST~eleven : thirty EST
+31/31/100~thirty one/thirty one/one hundred
+1-8090~one - eight thousand ninety
+f++~f plus plus
+dfsd#sfgsfd$~dfsd hash sfgsfd dollar
+dfsd$$sfgsfd$~dfsd dollar dollar sfgsfd dollar
+$12@12%~dollar twelve at twelve percent sign
+@12~at twelve
+12@~twelve at
+@mypizza~at mypizza
+#mytext#~hash mytext hash
+test$~test dollar
+covid-19~covid-nineteen
+55-millimeters long~fifty five-millimeters long
```

Comment:
will #me2 or #me-too be normalized? not that it needs to be, just wondering what the extension of serial is

Reply:
yes, this is also supported